skrf/plotting.py

```diff
@@ -757,7 +757,8 @@ def setup_matplotlib_plotting():
     networkSet.NetworkSet.plot_uncertainty_bounds_s = plot_uncertainty_bounds_s
     networkSet.NetworkSet.plot_logsigma = plot_logsigma
     networkSet.NetworkSet.signature = signature
-
+    
+    circuit.Circuit.plot_graph = plot_circuit_graph
```

Comment:
@dvincentwest can you take a look at this plotting architecture? i think it is as you  intended, but i cant remember the logic.